semantic_router/indices/pinecone.py

```diff
@@ -0,0 +1,69 @@
+from pydantic import BaseModel, Field
+import os
+import pinecone
+from typing import Any, List, Tuple
+from semantic_router.indices.base import BaseIndex
+import numpy as np
+
+class PineconeIndex(BaseIndex):
+    index_name: str
+    dimension: int = 768
+    metric: str = "cosine"
+    cloud: str = "aws"
+    region: str = "us-west-2" 
+    pinecone: Any = Field(default=None, exclude=True)
```

Comment:
Maybe we can call this `client` — we have a lot of `pinecone` variables and it is a little confusing

Reply:
Thanks for sorting this James.